### ISSUE TITLE
Notch issues

### ISSUE THREAD

**Issue by gedi-t** [dwarvesf/hidden#267]
### MacBook Air 15" 2013 notch issues.

On the new MacBooks there is a notch on the screen, when opening collapsed icons, not all of them show up and some of them hide behind the notch and cannot be clicked. 

<img width="420" alt="image" src="https://github.com/dwarvesf/hidden/assets/19917883/46650320-9993-47d9-82c1-6029fa39c840"> Laptop screen, with the notch

![image](https://github.com/dwarvesf/hidden/assets/19917883/e6d757d7-d217-466f-954a-e87a265f3c0a) extended screen

**Comment by eestein** [comment]
I've also noticed that, since day one, but since I usually use my macbook as a secondary screen that doesn't bother me much. But this week I'm travelling and have to use the macbook as the main screen and it's annoying :D 

bartender fixes that by creating a hover box that displays all icons below the bar and, consequently, below the notch as well. Just an idea that might be worth embracing :)

I'd like to take this opportunity to thank the creator(s) for doing a very nice job on the app!

**Comment by vurtomatic** [comment]
Yes, getting around the notch will make this perfect and I can ditch Bartender.

**Comment by OldhamMade** [comment]
The most recent version of Bartender on my machine swaps the entire set of "always shown" items with the hidden items. This behaviour is quite nice, and I think would be a good thing to replicate in Hidden. It _should_ get around the Notch issues.

**Comment by cberggren** [comment]
Agreed, this would be the one feature the app is missing.

**Comment by electriquo** [comment]
https://github.com/zkondor/znotch to the rescue :)

**Comment by Sufiane** [comment]
> https://github.com/zkondor/znotch to the rescue :)
Not the best solution as it change the height of the screen and does not work like Bartender, but still a good free work around, thanks for sharing !

Unfortunately i've seen no activities on this repo this quiet some time so i'm not sure we'll ever see a fix happening :(

**Comment by stewartadam** [comment]
Duplicate of #225

**Comment by electriquo** [comment]
[just use zNotch](https://github.com/dwarvesf/hidden/issues/267#issuecomment-1865796827)
